e2e/provision/init.sh

```diff
@@ -73,7 +73,7 @@ BRANCH=${NEPHIO_BRANCH:-$(get_metadata nephio-test-infra-branch "main")}
 NEPHIO_USER=${NEPHIO_USER:-$(get_metadata nephio-user "${USER:-ubuntu}")}
 NEPHIO_CATALOG_REPO_URI=${NEPHIO_CATALOG_REPO_URI:-$(get_metadata nephio-catalog-repo-uri "https://github.com/nephio-project/catalog.git")}
 K8S_CONTEXT=${K8S_CONTEXT:-"kind-kind"}
-K8S_VERSION=${K8S_VERSION:-"v1.29.2"}
+K8S_VERSION=${K8S_VERSION:-"v1.31.0"}
```

Comment:
It would be great if you can update at https://github.com/nephio-project/test-infra/blob/main/e2e/provision/playbooks/roles/bootstrap/defaults/main.yml#L27 ...what are your thoughts?

Reply:
Absolutely. Good catch.